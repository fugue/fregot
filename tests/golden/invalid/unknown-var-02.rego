package fregot.tests.invalid.unkown_var_02

test_kv[k] = v {
  # k is not assigned here.
  v = 1
} {
  # v is not assigned here.
  k = 2
} {
  # ok.
  v = 1
  k = 1
}
