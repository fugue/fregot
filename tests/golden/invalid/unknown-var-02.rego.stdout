passed: 0, failed: 0, errored: 1
fregot.tests.invalid.unkown_var_02.test_kv:
fregot ([34meval error[0m):
  "unknown-var-02.rego" (line 3, column 1):
  unknown variable:

    3| test_kv[k] = v {
       [31m^^^^^^^^^^^^^^[0m

  Unkown variable: k_0

  Stack trace:
    rule [33mtest_kv[0m at tests:1:1
